src/GitHub.VisualStudio.UI/Views/Dialog/RepositoryCreationView.xaml

```diff
@@ -107,17 +107,21 @@
                           Grid.Column="1"
                           MaxLength="{x:Static ghfvs:Constants.MaxRepositoryNameLength}"
                           Text="{Binding RepositoryName, UpdateSourceTrigger=PropertyChanged}" 
-                          SpellCheck.IsEnabled="True"
```

Comment:
It looks like spell checking has been disabled. Was this by design?

Reply:
Yea, I disabled it, when I couldn't type `VisualStudio` without it calling that an error, I decided that it was not useful...

Reply:
Lol, makes sense. Though I'm perfectly capable of misspelling the name of my repo. 😉

Reply:
It look like dotcom also has spell checking disabled fro this field.

![image](https://user-images.githubusercontent.com/11719160/55642011-34f39d80-57c7-11e9-8d1b-dcf7c7ccd70f.png)

Reply:
ok